.github/workflows/inspektor-gadget.yml

```diff
@@ -92,16 +124,6 @@ jobs:
         kubernetes version: 'v1.18.2'
         github token: ${{ secrets.GITHUB_TOKEN }}
 
-    - uses: azure/docker-login@v1
-      with:
-        username: ${{ secrets.CONTAINER_REGISTRY_USERNAME }}
-        password: ${{ secrets.CONTAINER_REGISTRY_PASSWORD }}
-
-    # TODO: how to avoid pushing a container before running integration tests
```

Comment:
Was the TODO fixed? :stuck_out_tongue:

Reply:
No, it was not really.

I re-added it but at the right place. PTAL.